src/integrations/sveltekit/config.ts

```diff
@@ -0,0 +1,110 @@
+import type { ResolvedConfig } from 'vite'
+import type { ManifestTransform } from 'workbox-build'
+import type { BasePartial, GlobPartial, RequiredGlobDirectoryPartial } from 'workbox-build/src/types'
+import type { SvelteKitVitePluginOptions, VitePWAOptions } from '../../types'
+
+type WorkboxConfig = Partial<BasePartial & GlobPartial & RequiredGlobDirectoryPartial>
+
+export function configureSvelteKitOptions(viteOptions: ResolvedConfig, options: Partial<VitePWAOptions>) {
+  const {
+    base = viteOptions.build.base ?? '/',
+    adapterFallback,
+  } = options.svelteKitVitePluginOptions ?? {}
+
+  // exclude pwa plugin from SSR build
+  if (typeof options.includeManifest === 'undefined')
+    options.includeManifest = 'client-build'
```

Comment:
I would expect the PWA plugin should only ever be used on a client build. Could that be the default or does there even need to be an option for it?

Reply:
it isa new option, included in this PR:
```ts
  /**
   * By default, the `manifest` will be included on the service worker's *precache*.
   *
   * @default true
   */
  includeManifest?: boolean | 'ssr-build' | 'client-build'
```

Reply:
right. what I'm saying is I don't think any user will ever set it to `ssr-build`, so maybe we don't need the option? it should always be done for the client build only

Reply:
done